ProfileAPI/ProfileAPI/Controllers/ProfileController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using ProfileAPI.Models;
+using ProfileAPI.Services;
+
+namespace ProfileAPI.Controllers
+{
+    [Route("[controller]")]
```

Comment:
For the ConferenceService this was "api/[controller]". Is it needed here as well?